ui/src/App.vue

```diff
@@ -130,6 +132,7 @@ export default {
         this._data.suggestedPosition = res.SuggestedPosition;
         this._data.suggestionTimer = res.SuggestionTimer;
         this._data.winner = res.Winner;
+        this._data.invalidMoves = res.InvalidMovesForCurrentPlayer || [];
```

Comment:
the `|| [ ]` shoudln't be necessary I think. Because server will always return an array, even if it is empty

Reply:
I guess

Reply:
But somehow, when I remove it then it is impossible to play as stones wont be placed